frontend/src/metabase/notifications/DashboardSubscriptionsSidebar/DashboardSubscriptionsSidebar.jsx

```diff
@@ -481,7 +481,7 @@ class DashboardSubscriptionsSidebarInner extends Component {
       );
     }
 
-    if (shouldDisplayNewPulse(editingMode, pulses)) {
+    if (shouldDisplayNewPulse(editingMode, pulses) && !isEmbeddingSdk()) {
```

Comment:
Just added a simple check. Why would we let this subscription channel selections sidebar be rendered on the SDK 🤷 

<img width="339" height="309" alt="Image" src="https://github.com/user-attachments/assets/98c82572-9ad6-44b3-b2af-994892c71c81" />